app/controllers/helm_infrastructures_controller.rb

```diff
@@ -43,6 +45,10 @@ def update
         "to_attributes" => @data_attributes.slice(:helm_cluster_template_id, :override_values, :is_active, :use_k8s_kibana)
       }
       broadcast(:app_group_updated, @helm_infrastructure.app_group.id)
+      
+      client = ArgoCDClient.new
```

Comment:
let's reuse the connection instead of creating the client again and again

Reply:
added in client in config initializers

Reply:
added client in config initializers